src/data/modifiers.txt

```diff
@@ -4350,7 +4350,7 @@ Item	iced-out bling	Familiar Weight: +5
 Item	icicle katana	Familiar Weight: +5
 # imitation whetstone: Makes Imitation Crab claws sharper
 Item	immense ballet shoes	Familiar Weight: +5
-Item	ironic moustache	Familiar Weight: +10, Effect: "+10 lbs."
+Item	ironic moustache	Familiar Weight: +10, Familiar Effect: "+10 lbs", Equips On: "Mini-Hipster"
```

Comment:
This one doesn't need the "Equips On" field, because `src/data/familiars.txt` specifies that the Mini-Hipster's default familiar equipment is the ironic moustache. The "Equips On" field is only required for /extra/ familiar equipment.

Reply:
Thanks, I'll remove it